chemprop/cli/hpopt.py

```diff
@@ -386,18 +384,21 @@ def main(args: Namespace):
         output_scaler = train_dset.normalize_targets()
         val_dset.normalize_targets(output_scaler)
         logger.info(f"Train data: mean = {output_scaler.mean_} | std = {output_scaler.scale_}")
+        output_transform = (
+            UnscaleTransform.from_standard_scaler(output_scaler) if output_scaler else None
+        )
```

Comment:
```suggestion
        output_transform = (
            UnscaleTransform.from_standard_scaler(output_scaler)
            if output_scaler is not None
            else None
        )
```
Make it consistent with the one in train.py (see [here](https://github.com/chemprop/chemprop/blob/v2/dev/chemprop/cli/train.py#L974-L978)).

Reply:
I was wondering if it is possible for the `output_scaler` to be `None` after running `train_dset.normalize_targets()`?

Reply:
I don't think so, `normalize_targets` should always return the scaler (see [here](https://github.com/chemprop/chemprop/blob/v2/dev/chemprop/data/datasets.py#L96))

Reply:
Good catch, I think changing train.py to `if output_scaler else None` is better because we just care if the output_scaler exists (if it is None vs False vs [], in all cases we want to not give it to UnscaleTransform). Do you have an opinion?

Reply:
Okay so it looks like we don't have to check if it is None?

Reply:
If it is never `None`, changing both to `output_transform = UnscaleTransform.from_standard_scaler(output_scaler)` should be fine.